cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ProductVersionDTO.java

```diff
@@ -131,6 +131,9 @@
     @ApiModelProperty( "The number of markups set for the product version" )
     private long markedUp;
 
+    @ApiModelProperty( "The number of sell-outs set for the product version" )
+    private long soldOut;
```

Comment:
rename

Reply:
it's the past tense of "sell-out", as `markedUp` is the past tense of "mark-up"

I read it like "how many versions have been `markedUp`/`soldOut`"